tests/test_parser.py

```diff
@@ -652,6 +652,14 @@ def test_parse_W(self):
             with pytest.raises(ParserError):
                 self.parser.parse(fmt, "W")
 
+    def test_parse_normalize_spaces(self):
+        assert self.parser.parse(
+            "Jun 1 2005  1:33PM", "MMM D YYYY H:mmA", normalize_spaces=True
+        ) == datetime(2005, 6, 1, 13, 33)
+        assert self.parser.parse(
+            "  \n Jun   1\t 2005\n ", "MMM D YYYY", normalize_spaces=True
+        ) == datetime(2005, 6, 1)
+
```

Comment:
We should add a test that checks this string will raise a parser error if `normalize_spaces=False`.

Reply:
Also it would be good to test this without a format string.